Extend nft_ethereum.native_mints to before block 14801608

UmerHA opened the pull request:
Brief comments on the purpose of your changes:
A small correction to PR 1709: Extend nft_ethereum.native_mints to before block 14801608.

*For Dune Engine V2*
I've checked that:
General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributors)

Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

dune-eng commented:
Workflow run id 3314510712 approved.

dune-eng commented:
Workflow run id 3314510713 approved.

soispoke commented:
In this case I think we have to start from the beginning, but yeah tests need to pass before we can look to merge anything

dune-eng commented:
Workflow run id 3315966008 approved.

UmerHA commented:
Hey @augustog, @soispoke

I fixed the two bugs:
- also removed block_number limit from test ✅
- implemented better duplication test for `ethereum.contracts` ✅ (due to a bug `ethereum.contracts` contains duplicates, ie same address has multiple namespaces. In PR 1709 I assumed they at least have a unique `created_at` value, which turned out to be false. Therefore a better duplication test was needed)

The tests run locally:
- unique col test: https://dune.com/queries/1454781 ✅
- custom test: https://dune.com/queries/1454797 ✅

So I believe the code is correct. The timeout makes sense to me, as removing the block number limit increases the blocks to process by ~15X (from 6.5% of all blocks to 100%,  see https://dune.com/queries/1454666).

Is there a way to remove or increase the timeout limit for this kind of PRs?

soispoke commented:
@UmerHA all good, needed to merge last changes from the main branch to get dbt slim ci running correctly, but everything lgtm now
